types/usb/index.d.ts

```diff
@@ -382,6 +382,12 @@ export interface Endpoint extends EventEmitter {
 
   /** Object with fields from the endpoint descriptor -- see libusb documentation or USB spec. */
   descriptor: EndpointDescriptor;
+
+  /**
+   * Clear the halt/stall condition for this endpoint.
+   * @param callback
+   */
+  clearHalt(callback: (error: Error) => void): void;
```

Comment:
Other functions return errors as strings. Does `clearHalt()` diverge from this standard or are the others wrong, perhaps?

Reply:
good point, I'm not entirely sure though, I can't get it to throw an error and I can't understand the native c code enough to determine that